service/templates/display_title.html

```diff
@@ -142,6 +142,14 @@ <h5 class="{% if loop.index != 1 %} expand-top-half {% endif %}">
 
       </div>
 
+      <div class="printhidden">
+        <h2 class="heading-small">What did you think of this service?</h2>
+        <p>
+          <a href="mailto:digital-register-feedback@digital.landregistry.gov.uk">Please give us your feedback</a> (takes 30 seconds).<br>
```

Comment:
I am thinking we should have the email address as an environment variable please.